tokio-test/Cargo.toml

```diff
@@ -23,6 +23,7 @@ async-stream = "0.3.3"
 
 bytes = "1.0.0"
 futures-core = "0.3.0"
+bstr = "1.6.2"
```

Comment:
The MSRV failure is because you are adding this dependency.

Reply:
Thanks! I'll take a look if it is possible to downgrade bstr. Otherwise i'll just copy paste* the relevant formatting code or cobble something together myself. 

That said, taking a step back, do you think this change is a good idea in the first place? 


* Checking the license first, of course

Reply:
You should not add a new dependency for this. But you can put together something simple yourself.